recipes/serialchemy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "serialchemy" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ef00f71153cc076709ed58ba00731768dbabbf15c237cd423df3c808b24a988
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - sqlalchemy >=1.1
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - sqlalchemy >=1.1
+
+test:
+  imports:
+    - serialchemy
+    - serialchemy._tests
+
+about:
+  home: http://github.com/ESSS/serialchemy
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Serializers for SQLAlchemy models.
+  doc_url: https://serialchemy.readthedocs.io/
+  dev_url: http://github.com/ESSS/serialchemy
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
+    - igortg
```

Comment:
@igortg is on vacation and probably won't be able to answer, so I suggest you remove it for now and we can add him later.

Reply:
Drop @igortg from maintainers for now